pkg/kubelet/deviceplugin/mock_device_plugin.go

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
I'm not familiar with the vocabulary difference here. I agree that fake is a better qualifier.
What does mock stand for ?